classes-recipe/image_types_tegra.bbclass

```diff
@@ -454,3 +454,4 @@ do_image_tegraflash[depends] += "${TEGRAFLASH_PKG_DEPENDS} dtc-native:do_populat
                                  virtual/bootloader:do_deploy virtual/secure-os:do_deploy ${TEGRA_SIGNING_EXTRA_DEPS} ${DTB_EXTRA_DEPS} \
                                  ${@'${TEGRAFLASH_INITRD_FLASH_IMAGE}:do_image_complete' if d.getVar('TEGRAFLASH_INITRD_FLASH_IMAGE') != '' else ''}"
 IMAGE_TYPEDEP:tegraflash += "${IMAGE_TEGRAFLASH_FS_TYPE}"
+DEPENDS += "${@'python3-pyyaml-native' if d.getVar('TEGRA_SIGNING_ARGS') != '' else ''}"
```

Comment:
It's probably better to just add this to DEPENDS in the tegra-flashtools-native recipe itself.  I wouldn't event bother with the TEGRA_SIGNING_ARGS conditional.

Reply:
Right, done!